subsys/mgmt/Kconfig.mcumgr

```diff
@@ -11,7 +11,12 @@ config MCUMGR
 
 if MCUMGR
 
-menu "Command handlers"
+menu "CBOR Attribute Library"
+config CBORATTR_MAX_SIZE
```

Comment:
would be great if key name will be in scope of mcumgr: MGMT_CBORATTR_MAX_SIZE etc